StringCalculatorKata/StringCalculator.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace StringCalculatorKata
+{
+    public static class StringCalculator
+    {
+        public static int Add(string numbers)
+        {
```

Comment:
please use Lists instead of Arrays
arrays heart my eyes :3 
it will be easier for u to deal with lists and use LINQ methods!